web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div>
+    <v-row class="mx-2 my-2">
+      <h1 class="font-weight-regular">
+        {{ meta.name }}
+      </h1>
+    </v-row>
+    <v-card>
+      <v-row
+        class="mx-2"
+        align-content="center"
+      >
+        <v-col>
+          <a :href="permalink">
+            {{ permalink }}
+          </a>
+        </v-col>
+        <v-tooltip
+          left
+          :disabled="editDisabledMessage === null"
+        >
+          <template v-slot:activator="{ on }">
+            <div v-on="on">
+              <v-btn
+                class="mt-2"
+                text
+                :disabled="editDisabledMessage !== null"
+                @click="$emit('edit')"
+              >
+                <v-icon class="mr-3 ml-0">
+                  mdi-pencil
+                </v-icon>
+                Edit metadata
+              </v-btn>
+            </div>
+          </template>
+          {{ editDisabledMessage }}
+        </v-tooltip>
+      </v-row>
+
+      <v-divider />
+
+      <v-row :class="titleClasses">
+        <v-card-title class="font-weight-regular">
+          Description
+        </v-card-title>
+      </v-row>
+      <v-row class="mx-1 px-4 font-weight-light">
+        {{ meta.description }}
+      </v-row>
+
+      <template v-for="(field, key) in extraFields">
+        <v-divider
+          :key="`${key}-divider`"
+          class="mt-4"
```

Comment:
This divider works well for maintaining spacing between list items, but the very last item doesn't have a divider, so it doesn't have a margin between it and the bottom border. It's purely cosmetic, but it looks very abreviated.